Assets/HoloToolkit/Input/Scripts/GazeStabilizer.cs

```diff
@@ -105,10 +105,10 @@ private void AddGazeSample(Vector3 positionSample, Vector3 directionSample)
                 // Remove from front if we exceed stored samples.
                 if (stabilitySamples.Count >= StoredStabilitySamples)
```

Comment:
On a side note, with this implementation SoredStabilitySamples can never be reduced below stabilitySamples.Count effectively. Using a while instead of an if would resolve this.